geonode/settings.py

```diff
@@ -23,13 +23,14 @@
 import os
 import sys
 import re
+from datetime import timedelta
 
 from kombu import Queue
 from geonode import __file__ as geonode_path
 from geonode import get_version
 from geonode.celery_app import app  # flake8: noqa
 from distutils.util import strtobool
-# import djcelery
+from django.conf import global_settings
```

Comment:
I think this may be redundant. As far as I know django always imports from global settings.

Reply:
Not exactly, `global_settings` provides defaults for all Django settings.  In this very case I wanted to get default `DATE_INPUT_FORMATS` and extend it with few other cases (instead of redefining)

Reply:
I see, would it be possible to export it explicitly instead? So we now is only importing that one and is a trace for future import if needed.